/*package demoOn16August2016;

import java.util.Scanner;

public class Exceptions {

	private static Scanner s;

	public static void main(String[] args) throws Exception {
		System.out.println("Input 2 numberical values :");
		try{
			s = new Scanner(System.in);
			int a = s.nextInt();
			int b = s.nextInt();
			System.out.println("Addition of 2 numbers :"+mfunc.add(a, b));
			System.out.println("Subtraction of 2 numbers :"+mfunc.sub(a, b));
			System.out.println("Multiplication of 2 numbers :"+mfunc.mul(a, b));
			System.out.println("Division of 2 numbers :"+mfunc.div(a, b));
		}catch(Exception e){
			System.out.println("2nd number should be not be zero");
		}
	}

}*/